lib.rs

```diff
@@ -101,6 +119,34 @@ The following features are experimental
 )]
 /*!
 
+With the `rustdoc` styling
+
+```rust
+#![doc = document_features::document_features!(feature_label = "<span class=\"stab portability\"><code>{feature}</code></span>")]
+```
+
+the generated documentation would look like the following:
+
+<table><tr><th>Preview</th></tr><tr><td>
+This comments goes on top
+
+* <span class="stab portability"><code>foo</code></span> *(enabled by default)* —  The foo feature enables the `foo` functions
+
+* <span class="stab portability"><code>bar</code></span> —  The bar feature enables the bar module
+
+#### Experimental features
+The following features are experimental
+* <span class="stab portability"><code>fusion</code></span> —  Enable the fusion reactor
+
+  ⚠️ Can lead to explosions
+
+#### Optional dependencies
+* <span class="stab portability"><code>genial</code></span> —  Enable this feature to implement the trait for the types from the genial crate
+
+* <span class="stab portability"><code>awesome</code></span> —  This awesome dependency is specified in its own table
+</td></tr></table>
```

Comment:
This should probably use the `self_test!` super-weapon, but I couldn't decipher how to use it.